README.md

```diff
@@ -114,6 +186,4 @@ When using OpenSMT as a library, the option needs to be set in `SMTConfig` **bef
 Interpolation is supported for SMT-LIB logics `QF_UF`, `QF_LRA`, and `QF_LIA` in both single-query and incremental mode. An example of how SMT-LIB2 file can be extended to instruct OpenSMT to compute interpolants can be found [here](regression_itp/itp_bug_small.smt2).
 
 ## Contact
-If you have questions please mail them to me at antti.hyvarinen@gmail.com, or at [github](https://github.com/usi-verification-and-security/opensmt)
-
-
+If you have questions, bug reports, or feature requests, please refer to our [GitHub](https://github.com/usi-verification-and-security/opensmt/issues) issue tracker or send us a mail at tomaqa@gmail.com or antti.hyvarinen@gmail.com.
```

Comment:
Since we are making changes here, can you add my email as well?
`martin.blicha@gmail.com`

Reply:
Sure. Should I also include you into the copyright at the top?

Reply:
And I suppose that `AUTHORS` should be updated too ..

Reply:
Yes that would be good as well.
I have never actually looked into the AUTHORS file :D
Maybe we can leave that to separate PR as well, so this one does not lose the focus on the Makefile.

Reply:
Done.